occrp/forms.py

```diff
@@ -11,13 +11,15 @@ class StoryForm(FlaskForm):
     title = StringField('title', validators=[DataRequired()])
 
 
+# Upon data entry: we should trim all strings, as they get entered
 class EventForm(FlaskForm):
-    title = StringField('title', validators=[DataRequired()])
-    start_date = StringField('start_date')
-    end_date = StringField('end_date')
-    description = StringField('description')
-    significance = StringField('significance')
-    person_name = StringField('person_name')
+    description = StringField('description', validators=[DataRequired()])
+    start_date = StringField('start_date', filters=[lambda x: x or None])
```

Comment:
These lambdas just ensure that we get a `NULL` in the database (instead of an empty string)?

Reply:
Yup, that's what they do. Now, we will not shove empty strings into the database. Addresses this issue: https://github.com/datamade/occrp-timeline-tool/issues/23